scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -1225,24 +1367,25 @@ public String getTaskServerLogsByTag(String jobId, String taskTag)
     @ImmediateService
     public Page<JobInfo> getJobs(int offset, int limit, JobFilterCriteria filterCriteria,
             List<SortParameter<JobSortParameter>> sortParameters) throws NotConnectedException, PermissionException {
-        UserIdentificationImpl ident = frontendState.checkPermission("getJobs",
-                                                                     "You don't have permissions to load jobs");
-
-        boolean myJobsOnly = filterCriteria.isMyJobsOnly();
+        synchronized (waitForJobsToBeSubmitted) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)